Update 05-profiling-your-MARC-data.md (#244)
LibraryCarpentry/lc-marcedit: episodes/05-profiling-your-MARC-data.md

Reviewer: ```suggestion
The results list includes two records with duplicate 245 fields, # 8 and # 28. Click on Jump to Record # to be taken to the record in the MarcEditor. Once on the record you can manually delete the duplicated 245 field.
```

::::::::::::::::::::::::::::::::::::::: objectives

- Understand and run reports for your MARC data
- Understand and run validation for your MARC data
- Understand where to find tools in the MARCedit Edit toolbar and shortcuts

> - Explain the difference between to Find and Find All

Reviewer: ```suggestion
- Explain the difference between Find and Find All
```

:::::::::::::::::::::::::::::::::::::::  challenge

> #### Find and correct the error with the title or the MARC field 245 using the Field Count Report

Reviewer: I think this exercise title should stay as it is

1. Describe the Field Count report results
2. Identify the error for at least one of the records for the title
3. Fix the error for at least one of the records for the title

:::::::::::::::  solution

#### Solution

1. Field Count outlines each field found in our records, the number of times it occurs, and how many records it occurs in. Field count can also tell us the number of records in our file (536).
2. The non-repeatable field 245 occurs more times than the number of records in the file. In other words, at least one record contains multiple 245 fields. Field 245 appears in one less record than other required fields.

> 3. Fix the missing MARC field 245 for the title 'McGregor Smith Scout Reservation Master Plan by Erik Schofield' by changing the line with =2 4 \0$a to =245  \0$a

Reviewer: ```suggestion
```

Reviewer: This exercise should only be identifying that there are general issues with the 245 fields, since we then show how to use other tools to actually identify and correct the issues below. If we correct the issues in this exercise, the next steps are not useful.

> 4. Fix the missing MARC field 245 for the title 'Fundo Mandorpampa Ecological Center by Timothy Christian Grey' by removing the duplicate MARC field 245.

Reviewer: ```suggestion
```

:::::::::::::::::::::::::::::::::::::::  challenge

#### Correct the missing 245 field

1. What is the issue with this record? Is the 245 formatted correctly?

:::::::::::::::  solution

#### Solution

> 1. The 245 field number is formatted incorrectly. The MarcEditor relies on a standard format to identify and take actions on fields. Remove the rogue space and update the field number to correct this record.

Reviewer: ```suggestion
1. The 245 field number is formatted incorrectly. The MarcEditor relies on a standard format to identify and take actions on fields.  Fix the missing MARC field 245 for the title 'McGregor Smith Scout Reservation Master Plan by Erik Schofield' by changing the line with =2 4 \0$a to =245  \0$a Removing the rogue space in the field number will correct this record.
```
